.travis.yml

```diff
@@ -1,6 +1,6 @@
 language: node_js
 node_js:
-- 5.6
+- stable
```

Comment:
This should actually be `node` instead of `stable`:
https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Specifying-Node.js-versions